application/libraries/Api/Command/V1/Transformer/Output/TransformerOutputQuestion.php

```diff
@@ -34,19 +34,24 @@ public function __construct()
         ]);
     }
 
-    public function transformAll($array)
+    public function transformAll($collection)
```

Comment:
This function looks like an exact copy of the previous transformAll but with another key to sort by? Guess it's OK for now. :)

Reply:
There is a trade off between code re-use and code complexity.